NavigateU/NavigateU/RegistrationScreen/RegistrationScreenView.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  RegistrationScreenView.swift
+//  NavigateU
+//
+//  Created by Faki Doosuur Doris on 19.03.2024.
+//
+
+import UIKit
+
+final class RegistrationScreenView: UIView {
+
+    lazy var appName: UILabel = UILabel()
+    lazy var firstName: UITextField = UITextField()
+    lazy var lastName: UITextField = UITextField()
+    lazy var email: UITextField = UITextField()
+    lazy var password: UITextField = UITextField()
+    lazy var countryOfOrigin: UITextField = UITextField()
+    lazy var cityOfResidence: UITextField = UITextField()
+    lazy var terms: UILabel = UILabel()
+    lazy var conditions: UILabel = UILabel()
+    lazy var registerButton: UIButton = UIButton()
```

Comment:
probably there is no reason to make an RegistrationScreenView, usually separate classes are made for displays that will be used in the whole application in one style, for example, a button, and all this can be placed in UIViewController, so it will be easier to access them